media/css/mozorg/home/brand-popup.scss

```diff
@@ -0,0 +1,111 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
+.brand-popup {
+    @include at2x('/media/img/home/2016/brand/popup-graphic.png', 718px, 254px);
+    background-color: #000;
+    background-position: right -350px center;
+    background-repeat: no-repeat;
+    position: relative;
+    width: 100%;
+
+    @media #{$mq-desktop} {
+        background-position: right -200px center;
+    }
+
+    @media #{$mq-desktop-wide} {
+        background-position: right  center;
+    }
+
+    .content {
+        position: static;
+    }
+
+    h2 {
+        @include font-size(34px);
+        color: #fff;
+        font-weight: bold;
+        margin-bottom: 20px;
+    }
+
+    .popup-cta {
+        @include font-size(18px);
+        color: #000;
+        display: inline-block;
+        line-height: 2.5;
+        position: relative;
+        text-decoration: none;
+        text-transform: uppercase;
+
+        span {
+            @include transition(background-color 150ms ease-out);
+            background-color: #ebc044;
+            padding: 10px 15px;
+            -webkit-box-decoration-break: clone;
+            box-decoration-break: clone;
```

Comment:
New to me too (found it on MDN) - it doesn't work in IE or Edge yet, but the degradation is probably acceptable.